examples/gateway/mlflow_serving/README.md

```diff
@@ -0,0 +1,181 @@
+
+> **NOTE:** The MLflow Model Server does not have to be running in order to update the configuration file or to start the MLflow AI Gateway. In order to respond to submitted queries, it is required to be running.
+
+
+## Creating and logging a fill mask model
+To support an additional route for generating a mask fill response from masked input text, we need to log an appropriate model. 
+For this tutorial example, we'll use a `transformers` `Pipeline` wrapping a `BertForMaskedLM` torch model and will log this pipeline using the MLflow `transformers` flavor.
+
+```python
+from transformers import AutoTokenizer, AutoModelForMaskedLM
+import mlflow
+
+
+lm_architecture = "bert-base-cased"
+artifact_path = "mask_fill_model"
+
+tokenizer = AutoTokenizer.from_pretrained(lm_architecture)
+model = AutoModelForMaskedLM.from_pretrained(lm_architecture)
+
+components = {
+    "model": model,
+    "tokenizer": tokenizer
+}
+
+with mlflow.start_run():
+    mlflow.transformers.log_model(
+        transformers_model=components,
+        artifact_path=artifact_path,
+    )
+    model_uri = mlflow.get_artifact_uri(artifact_path)
+
+# Generate the cli command for starting a local MLflow Model Serving endpoint for this fill mask model
+print(f"mlflow models serve -m {model_uri} -h 127.0.0.1 -p 9010 --no-conda")
+```
+
+## Starting the model server for the fill mask model
+Using the command printed to stdout from above, open a new terminal (do not close the terminal that is currently running the embeddings model being served!) 
+and paste the command.
+
+```commandline
+mlflow models serve -m file:///Users/me/demos/mlruns/0/bc8bdb7fb90c406eb95603a97742cef8/artifacts/mask_fill_model -h 127.0.0.1 -p 9010 --no-conda
+```
+
+## Update the config.yaml to add a new completions route
+Ensure that the Mlflow serving endpoint starts and is ready for traffic.
+
+```commandline
+2023/08/08 17:39:14 INFO mlflow.models.flavor_backend_registry: Selected backend for flavor 'python_function'
+2023/08/08 17:39:14 INFO mlflow.pyfunc.backend: === Running command 'exec gunicorn --timeout=60 -b 127.0.0.1:9010 -w 1 ${GUNICORN_CMD_ARGS} -- mlflow.pyfunc.scoring_server.wsgi:app'
+[2023-08-08 17:39:15 -0400] [55722] [INFO] Starting gunicorn 20.1.0
+[2023-08-08 17:39:15 -0400] [55722] [INFO] Listening at: http://127.0.0.1:9010 (55722)
+[2023-08-08 17:39:15 -0400] [55722] [INFO] Using worker: sync
+[2023-08-08 17:39:15 -0400] [55722] [INFO] Booting worker with pid: 55724
+```
+
+Add the entry to the MLflow AI Gateway configuration file (the full file is shown below):
+
+```yaml
+routes:
+  - name: embeddings
+    route_type: llm/v1/embeddings
+    model:
+      provider: mlflow-model-serving
+      name: sentence-transformer
+      config:
+        model_server_url: http://127.0.0.1:9020
+  - name: fillmask
+    route_type: llm/v1/completions
+    model:
+      provider: mlflow-model-serving
+      name: fill-mask
+      config:
+        model_server_url: http://127.0.0.1:9010
+```
```

Comment:
Why not putting a lint to file examples/gateway/mlflow_serving/config.yaml here ?